examples/push/BUILD.bazel

```diff
@@ -40,8 +55,8 @@ stamp_tags(
 oci_push(
     name = "push_image_index",
     image = ":image_index",
-    repository = "index.docker.io/<ORG>/image",
-    repotags = ":stamped",
+    repository_file = ":stamped_repo",
```

Comment:
ah, because we can't detect on the attribute type this time. The other ones use a trick in the macro like `types.is_list` or `types.is_dict`. But a string could be a label or it could be a repository name and the macro can't tell.

Reply:
this should be possible now that we have  unified  `repotags` which is a list